### ISSUE TITLE
Bad export

### ISSUE THREAD

**Issue by YAGrand** [dabeng/OrgChart#336]
Hi, I try to export my chart to png, as a result I get cropped image.
![mychart](https://user-images.githubusercontent.com/11403132/34827938-e25c80a0-f6ec-11e7-9677-08682b9f2921.png)

**Comment by dabeng** [comment]
Hi @YAGrand , this is a [huge orgchart](https://codepen.io/dabeng/pen/eRXpbW) both in horizontal direction or vertical direction. Please try to export it to the png picture again.

**Comment by YAGrand** [comment]
Normal export occurs only if the main container position = initial

**Comment by YAGrand** [comment]
I solved the problem by adding directives to the css

```
#chart-container.canvasContainer{
  position: initial;
}
```

**Comment by YAGrand** [comment]
New problem. How to make a picture sharper?
![unsharp_chart](https://user-images.githubusercontent.com/11403132/34876804-4d021254-f7b4-11e7-9c68-ffae192cfec9.png)

**Comment by scrobbleme** [comment]
@YAGrand Try the PDF export. Still an embedded image, but works much better for me than PNG.

From the doc: 

> (3) Besides, if you wanna export a pdf format or your orgchart includes picture, you have to introduce jspdf and set "exportFileextension" option to "pdf".

**Comment by YAGrand** [comment]
Tried, but not exported to pdf. See gif:
![nopdfexport](https://user-images.githubusercontent.com/11403132/34931192-72e3cd92-f9de-11e7-89f5-454e09ade680.gif)

**Comment by scrobbleme** [comment]
You are right. Exporting is not soo nice. You may try another font, avoid rounded corners...  But I think it is not orgChart to blame. It is propably the browser and html2canvas... (btw. are you already using the latest [1.0.0-a9 of html2canvas](https://github.com/niklasvh/html2canvas/releases)?)

Another way, you maybe can try and report back: use an html2pdf or html2image converter at the backend. If you are using PHP, you may try this one: https://html2pdf.fr/en/home

**Comment by YAGrand** [comment]
Using **html2canvas 0.5.0-beta3**
Tried **html2canvas 1.0.0-alpha.9**, not woking pdf and png export. 

> Another way, you maybe can try and report back: use an html2pdf or html2image converter at the backend. If you are using PHP, you may try this one: https://html2pdf.fr/en/home

Does it support the transform property?

**Comment by scrobbleme** [comment]
Strange, I'm using html2canvas 1.0.0 a9, with jspdf 1.3.5 and orgChart 2.0.15. (in Opera, Chrome, Firefox, Edge it is working for me)

Don't knwo if it is supporting transform, maybe just try ;)

**Comment by todorivanov** [comment]
@YAGrand maybe you should check the browser console. Are there any errors? Sometimes, when the image is too large, the browser may not be able to transform it to canvas. Check and see if there are any errors.

**Comment by dabeng** [comment]
Hi @YAGrand , feel free to play around this **[Demo](https://codepen.io/dabeng/pen/Pzoxga)** and put forward your new question.
